code/modules/gear_presets/synth_k9.dm

```diff
@@ -0,0 +1,38 @@
+/datum/equipment_preset/synth_k9/
+	name = "K9 Synthetic"
+	uses_special_name = TRUE
+	languages = ALL_SYNTH_LANGUAGES
+	skills = /datum/skills/synth_k9
+	minimap_icon = "synth_k9"
+	flags = EQUIPMENT_PRESET_START_OF_ROUND|EQUIPMENT_PRESET_MARINE
+	faction = FACTION_MARINE
+	idtype = /obj/item/card/id/gold
+	assignment = JOB_SYNTH_K9
+	rank = "Synthetic K9"
+	paygrade = PAY_SHORT_SYN_K9
+	role_comm_title = "K9"
+
+/datum/equipment_preset/synth_k9/New()
+	. = ..()
+	access = get_access(ACCESS_LIST_GLOBAL)
+
+/datum/equipment_preset/synth_k9/load_race(mob/living/carbon/human/new_human)
+	. = ..()
+	new_human.h_style = "Bald"
+	new_human.f_style = "Shaved"
+	if(new_human.client?.prefs?.synthetic_type)
+		new_human.set_species(new_human.client.prefs.synthetic_type)
+		return
+	new_human.set_species(SYNTH_K9)
```

Comment:
so cursed lmao

Reply:
You're not wrong lmao, its the nature of half our game being about humans.

Reply:
To specify: the hair never comes up during the creation of this specific mob.  But on gib where we create individual body parts we then reconstruct the head from scratch and add what wasn't there.  In this case, we'd put the default human hair onto the head of a mob that never had it in the first place.